cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,15 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('https://react-redux.realworld.io');
+    cy.contains('Sign up').click();
+    cy.get('input[placeholder="Username"]').type(user.username);
+    cy.get('input[placeholder="Email"]').type(user.email);
+    cy.get('input[placeholder="Password"]').type(user.password);
+    cy.get('button[type="submit"]').click();
+    cy.get('.navbar').should('contain', user.username);
```

Comment:
After submitting the form, it would be good to also check if the user has been redirected to the expected page. You can do this by checking the URL or the presence of certain elements that only exist on the expected page.